core/store/src/columns.rs

```diff
@@ -444,7 +450,8 @@ impl DBCol {
             | DBCol::StateChangesForSplitStates
             | DBCol::StateHeaders
             | DBCol::TransactionResultForBlock
-            | DBCol::Transactions => true,
+            | DBCol::Transactions
+            | DBCol::ShardUIdMapping => true,
```

Comment:
Can you explain how will the mapping work on split storage nodes? I can't say if this is good or not without the full picture. For early MVP it doesn't matter too much so feel free to just leave a TODO here and proceed.

Reply:
My understanding was we want to copy `ShardUIdMapping` to cold storage because `cold_store.get(state_key)` would not work otherwise. Marked it as TODO do understand it deeper after early MVP.